### PR TITLE
Show Total Staked in USD instead of AVAX in Farm page

### PR DESCRIPTION
**Pull request by SarjuHansaliya**

### CODE REVIEW

File: src/state/stake/hooks.ts [hunk 1]
```diff
@@ -893,6 +904,7 @@ export interface DoubleSideStakingInfo extends StakingInfoBase {
   multiplier: JSBI
   // total staked AVAX in the pool
   totalStakedInWavax: TokenAmount
+  totalStackedInUsd: TokenAmount
```
Comment (bmino) [inline src/state/stake/hooks.ts:907]: Lil typo haha. Should be `totalStakedInUsd`

File: src/pages/Earn/Manage.tsx [hunk 2]
```diff
@@ -230,7 +229,7 @@ export default function Manage({
           <AutoColumn gap="sm">
             <TYPE.body style={{ margin: 0 }}>{t('earnPage.totalStaked')}</TYPE.body>
             <TYPE.body fontSize={24} fontWeight={500}>
-              {`${valueOfTotalStakedAmountInWavax?.toSignificant(4, { groupSeparator: ',' }) ?? '-'} AVAX`}
+              {`${valueOfTotalStakedAmountInUSDC?.toSignificant(4, { groupSeparator: ',' }) ?? '-'} USD`}
```
Comment (bmino) [inline src/pages/Earn/Manage.tsx:232]: Would it be better to use a `$` symbol here instead of `USD`?

**Review by bmino — COMMENTED** [review]
We are still sorting by wavax in a few places like the `Earn/index.tsx` which will work fine since the wavax and usd values should be ordered the same, but would it make sense to sort on usd in all cases where available for consistency?

File: src/state/stake/hooks.ts (hunk 1 above)
Comment (SarjuHansaliya) [reply src/state/stake/hooks.ts:907]: haha, will update it.

File: src/pages/Earn/Manage.tsx (hunk 2 above)
Comment (SarjuHansaliya) [reply src/pages/Earn/Manage.tsx:232]: sure

Comment (SarjuHansaliya) [reply src/pages/Earn/Manage.tsx]: resolved

File: src/state/stake/hooks.ts (hunk 1 above)
Comment (SarjuHansaliya) [reply src/state/stake/hooks.ts]: resolved

**Review by bmino — APPROVED** [review]

**Pull request merged**